epytope/EpitopePrediction/External.py

```diff
@@ -6538,9 +7864,12 @@ def parse_external_result(self, file):
                 if row[0] in ["#", "-"] or row.strip() == "" or "pos" in row:
                     continue
                 else:
-                    allele, pep, score = Allele(row.split()[HLAIndex.PICKPOCKET_1_1]), Peptide(row.split()[PeptideIndex.PICKPOCKET_1_1]), float(row.split()[ScoreIndex.PICKPOCKET_1_1])
+                    allele = row.split()[HLAIndex.PICKPOCKET_1_1].replace('*','')
```

Comment:
Why is the replacement of `*` necessary now?

Reply:
Yeah this is an ugly one by PickPocket.. That is because the allele input representation (`HLA-A0101`) differs from the allele output representation (`HLA-A*0101`). Since I save the allele input representation with the `Allele Obj` in a dictionary in `predict`, it needs to be changed to the allele input representation